### ISSUE TITLE
Horizontal placement of ~ 

### ISSUE THREAD

**Issue by IceMan81** [be5invis/Iosevka#36]
Tilde's placement looks too high. See screenshot. Wouldn't it look better if it is centered?

<img width="656" alt="screen shot 2015-12-09 at 5 16 33 pm" src="https://cloud.githubusercontent.com/assets/1093778/11704107/9e7d54ca-9e99-11e5-8988-cd7e90651674.png">

**Comment by IceMan81** [comment]
Another area where the horizontal placement looks better centered.
![tilde_placement](https://cloud.githubusercontent.com/assets/1093778/11723209/f315c976-9f21-11e5-990e-8fb0df8d53ca.png)

**Comment by bitserf** [comment]
Hi @IceMan81, 

I did a local patch which moves tilde down. 

It is probably an ugly way to do it, but it works for me:

```diff
diff --git a/glyphs/overmarks.ptl b/glyphs/overmarks.ptl
index be1afb2..4a163fd 100644
--- a/glyphs/overmarks.ptl
+++ b/glyphs/overmarks.ptl
@@ -11,6 +11,9 @@ local markDotsRadius : DOTRADIUS * markStress / HALFSTROKE
 local aboveMarkTop (XH + ACCENT * 1.38)
 local aboveMarkBot (XH + ACCENT * 0.35)

+local middleMarkTop [MIDDLE + ACCENT * 1.38]
+local middleMarkBot [MIDDLE + ACCENT * 0.35]
+
 local belowMarkBot (0 - ACCENT * 1.38)
 local belowMarkTop (0 - ACCENT * 0.35)

@@ -103,8 +106,8 @@ symbol-block 'Above marks'
 		local leftEnd (markMiddle - markExtend * 1.5)
 		local rightEnd (markMiddle + markExtend * 1.5)

-		local ttop aboveMarkTop
-		local tbot : aboveMarkBot + markFine / 2
+		local ttop middleMarkTop
+		local tbot : middleMarkBot + markFine / 2
 		local top : ttop + (markFine * 2)
 		local bot : tbot - (markFine * 2)
```

It looks like this:
<img width="628" alt="tildesample" src="https://cloud.githubusercontent.com/assets/167841/11912419/b33e111a-a6a0-11e5-9a70-89c565d9feb1.png">

I also tweaked underscore to be below the baseline:

```diff
diff --git a/glyphs/symbol-punctuation.ptl b/glyphs/symbol-punctuation.ptl
index 26c4c61..12de333 100644
--- a/glyphs/symbol-punctuation.ptl
+++ b/glyphs/symbol-punctuation.ptl
@@ -410,7 +410,7 @@ symbol-block 'Dashes'
 		set-width WIDTH
 		assign-unicode '_'

-		include : HBarBottom SB RIGHTSB 0
+		include : HBarBottom SB RIGHTSB DESCENDER

 	create-glyph 'overline' : glyph-construction
 		assign-unicode 0x203E
```

It looks like this:

<img width="213" alt="underscoresample" src="https://cloud.githubusercontent.com/assets/167841/11912425/de68f242-a6a0-11e5-87cd-d37daf34dc15.png">


Maybe this is useful.

**Comment by IceMan81** [comment]
Neat! @bitserf Why don't you submit separate PRs for your change to tilde's position and for the underscore change. @be5invis can pull it in to his repo if he is in agreement.

**Comment by be5invis** [comment]
@bitserf I will provide a centered variant of tildes. It can be implemented like this:

```
create-glyph 'asciitilde.center' : glyph-construction
    include glyphs.asciitilde
    include : Upright
    include : Translate 0 (parenMid - XH - ACCENT)
    include : Italify
```
The undefline can be handled in the similar way. So, add it into 1.5?

**Comment by bitserf** [comment]
@be5invis sounds good to me, i look forward to 1.5. will variant be chosen by overriding STYLE_* variable when building?

**Comment by be5invis** [comment]
@bitserf Well the new glyph ITSELF is a stylisitic variant, and you can actually select it using selectors.  Opentype features about it will be provided either.

**Comment by IceMan81** [comment]
@be5invis Look forward to this.

**Comment by be5invis** [comment]
@IceMan81 @bitserf In 1.5 I will provide three Opentype features:

* `cv17`: Center the /asciitilde (`~`)
* `cv19`: Center the /asterisk (`*`)
* `ss10`: Center both of them

... while `cv16`, `cv18` and `ss09` push them back upward (because maybe you will change the style selector and center them by default).

![image](https://cloud.githubusercontent.com/assets/240091/11967923/d20613b0-a941-11e5-913f-93027e186619.png)
![image](https://cloud.githubusercontent.com/assets/240091/11967926/d723ecaa-a941-11e5-9c84-6c5f7062bcde.png)

**Comment by IceMan81** [comment]
@be5invis Any way I could generate it as default instead of to use in editors that do not support open type like Intellij and sublime? 

Btw, appreciate all the work you are doing creating this font. Pretty awesome!!

**Comment by be5invis** [comment]
@IceMan81 I will add some new styles into it.

**Comment by be5invis** [comment]
@IceMan81 Now using the lasted source code you can build them by this:
```bash
make STYLE_COMMON='v-tilde-low v-asterisk-low'
```

**Comment by bitserf** [comment]
Ahh, that's awesome! Thanks @be5invis! Less local patches to maintain :)

**Comment by IceMan81** [comment]
@be5invis Excellent!! I'll build with this.

**Comment by be5invis** [comment]
Released in 1.5.0. Closed.
